trulens_eval/examples/snowflake_demo/common_ui.py

```diff
@@ -133,10 +133,10 @@ def login():
 
 def get_tru_app_id(
     model: str, temperature: float, top_p: float, max_new_tokens: int,
-    use_rag: bool
+    use_rag: bool, retrieval_filter: float
 ):
     # Args are hashed for cache lookup
-    return f"app-prod-{model}{'-rag' if use_rag else ''} (temp-{temperature}-topp-{top_p}-maxtokens-{max_new_tokens})"
+    return f"app-prod-{model}{'-rag' if use_rag else ''}-retrieval-filter-{retrieval_filter} (temp-{temperature}-topp-{top_p}-maxtokens-{max_new_tokens})"
```

Comment:
if you don't use a RAG the retriever filter shouldn't matter no?

Reply:
good point - updated